Content.Server/Botany/Components/SeedComponent.cs

```diff
@@ -15,6 +15,12 @@ public sealed partial class SeedComponent : SharedSeedComponent
         [DataField("seed")]
         public SeedData? Seed;
 
+        /// <summary>
+        ///     The health value of a plant resulting from this seed. If null defaults to the plant's Endurance.
```

Comment:
```suggestion
        ///     If not null, overrides the plant's initial health. Otherwise, the plant's initial health is set to the Endurance value.
```